buildroot-external/busybox.config

```diff
@@ -951,7 +951,7 @@ CONFIG_PING6=y
 # CONFIG_ROUTE is not set
 # CONFIG_SLATTACH is not set
 # CONFIG_SSL_CLIENT is not set
-CONFIG_TC=y
+CONFIG_TC=n
```

Comment:
Can't remember the error this was causing but I don't think it's needed

>TURBOchannel is a DEC (now Compaq (now HP)) bus for Alpha and MIPS
processors. TURBOchannel programming specifications are available

Reply:
BTW: There is no possibility to specify`=n`for excluding any `CONFIG_XXXX` config option. If you need to ensure `CONFIG_TC` is not set you have to explicitly add a `# CONFIG_TC is not set` line instead.